SPECS/vim/CVE-2024-43802.patch

```diff
@@ -0,0 +1,46 @@
+From 322ba9108612bead5eb7731ccb66763dec69ef1b Mon Sep 17 00:00:00 2001
```

Comment:
Please add the note for what you changed from the upstream patch.

Reply:
note added